Customize with CSS/Readme.md

```diff
@@ -1,61 +1,79 @@
 
-One of the main components of WebVIs is a html panel that is used to display HTML5 custom elements, that can be connected to the block diagram. One of the main advantages of leveraging the styling to the web is the separation of concerns. Meaning, a customer can create a custom CSS file that can be shared across the organization or even among other users to build more and more styles and layouts.
+One of the main parts of a WebVI is an HTML output that displays HTML5 Custom Elements. One of the advantages of using CSS in a web application is to the separate content from style. For example, you could create a custom CSS file that is shared across the organization or even among other users to build more and more styles and layouts.
+
+# Usage
+
+You can use this approach in your own projects if you are familiar with CSS and how it interacts with HTML. For example, you can customize controls and indicators to match a corporate color scheme or make them accessible for color-blind users or in low contrast situations.
 
 # Dependencies
 - LabVIEW NXG 2.0 Beta
 - LabVIEW 2009-2017 (Required for hosting only)
 
 # Setup
-- Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repo to your machine.
-- Open `Customize with CSS/Customize with CSS.lvproject`
-    - Run the Web VI
-        - Open Main.gviweb and click the Run button
-    - Build Web application
-        - Open WebApp.gcomp
-        - Switch to the Document tab
-        - Click Build
+1. Clone the [ni/webvi-examples](https://github.com/ni/webvi-examples) repository to your machine.
+2. Open `Customize with CSS/Customize with CSS.lvproject`.
+3. On the **Project Files** tab, expand `WebApp.gcomp` and open `Main.gviweb`.
+4. Click the **Run** button.
+5. Build the web application.
+  a. On the **Project Files** tab, double-click `WebApp.gcomp` to open the web application component.
+  b. On the **Document** tab, click **Build**.
+**Note:** To view the build output on your machine, click **Locate Directory in Windows Explorer** on the **Document** tab once your application finishes building.
 
 # Hosting
 
-Place the entire contents of the `Customize with CSS/Builds/Web Server/Configuration1/WebApp` directory within any Web server of your choosing.
+Copy and paste the entire `Customize with CSS/Builds/Web Server/Configuration1/WebApp` directory to any web server you want.
 
 ## Hosting with LabVIEW 2009-2017 Web Server
 
-1. Open `C:\Program Files (x86)\National Instruments\Shared\NI WebServer\www`
-2. Copy `Customize with CSS/Builds/Web Server/Configuration1/WebApp` directory into the `www` directory
-3. Open a Web Browser and navigate to `http://localhost:8080/WebApp/index.html`
+1. Open `C:\Program Files (x86)\National Instruments\Shared\NI WebServer\www`.
+2. Copy `Customize with CSS/Builds/Web Server/Configuration1/WebApp` directory into the `www` directory.
+3. Open a web browser and navigate to `http://localhost:8080/WebApp/index.html`.
```

Comment:
Needs 2 spaces at the end of the line to preserve the newline before the **Note:**